plugwise_usb/network/cache.py

```diff
@@ -77,3 +77,14 @@ async def update_nodetypes(self, mac: str, node_type: NodeType | None) -> None:
     def get_nodetype(self, mac: str) -> NodeType | None:
         """Return NodeType from cache."""
         return self._nodetypes.get(mac)
+
+    async def prune_cache(self, registry: list[str]) -> None:
+        """Remove items from cache which are not found in registry scan."""
+        _new_nodetypes: dict[str, NodeType] = {}
```

Comment:
Underscore is not useful for a local variable.

Only useful for self's or functions. Using an underscore means making the self or function private to the file its in.